content/pages/volunteers.md

```diff
@@ -0,0 +1,96 @@
+Title: Volunteer Opportunities
+Category: pages
+Summary: Volunteer opportunities available with WiCS undergraduate.
+
+
+As we strive to improve the quality of our events, every contribution, large or
+small counts. Volunteering with our committee is a great way to get involved
+with us on a smaller scale. While being a committee member requires you to
+dedicate at least 8 hours per week to us, becoming a core volunteer is an
+easier, lower commitment way to be involved in our activities, help us run
+events and workshops, and get to know the committee members.  Volunteering is
+also a good way to figure out if you'd like to commit more time to WiCS and
+eventually apply to be a committee member.
+
+We have a range of volunteer positions, each of which will help you to learn
+about an aspect of our committee. By volunteering to be a **photographer**, you
+can learn about how we run events and help with publicizing them. By
+volunteering to be a **technical mentor** for our events, you can help teach
+technical concepts to others. By becoming a **wics-sys volunteer**, you can
+learn systems administration and static website development from our systems
+committee.
+
+## How volunteering works
+
+You can sign up for most positions by filling out
+[this](http://goo.gl/forms/NsOXea6Yjf5t02Oj1) form. The only exception is
+volunteering to run your own event (mentioned below), in which case you'll have
+to [email us directly](mailto:wics-ugrad@lists.uwaterloo.ca). You will be added
+to our volunteer mailing list and be notified of future volunteer opportunities
+by email. Most volunteer opportunities will be assigned on a
+first-come-first-serve basis. However, if you have volunteered for the same
+position at least twice in a row, someone else will be given the opportunity so
+that everyone gets to do something! Every 1-2 weeks, an email will be sent out
+to the mailing list with upcoming volunteer opportunities. If any of the
+opportunities interest you, verify that you are free on that day and reply back
+to the email.
+
+Here are the volunteer positions currently available.
+
+### Photographer
+
+* Each event will have a main photographer and a back up photographer. In case
+the main photographer has a last minute emergency, they should contact the
+committee immediately so that the backup can be informed.
+* **You will need:** A camera of your own or one that you can borrow.
+* People who are not willing to be photographed will be in lanyards so make sure that you don’t photograph them!
+* Photographs should be sent to the committee as soon as possible after
+ the event is over.
```

Comment:
Markdown style thing: if you have a line wrap around on something indented like an enumeration here, make sure to indent the following lines so they line up with the rest of the text. "the" should be one more space indented. (this comment applies throughout)